src/status_im/ui/screens/wallet/choose_recipient/events.cljs

```diff
@@ -17,22 +18,25 @@
              amount (assoc :amount amount))))
 
 (defn- extract-details
-  "First try to parse as EIP67 URI, if not assume this is an address directly.
-   Returns a map containing at least the `address` key"
-  [s]
-  (or (eip67/parse-uri s) {:address s}))
```

Comment:
To clarify, is EIP681 a strict superset of EIP67? Since we are changing behavior of this

Reply:
yes